README.md

```diff
@@ -1 +1,50 @@
-hello
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School Library
+> A web application to record what books are in the library and who borrows them. Built with: Ruby
+## Built With
+
+- Ruby
+
+## Requirements 
+Install:
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+- [IRB](https://github.com/ruby/irb)
+
+## Getting Started
+
+To get a local copy follow these simple example steps.  
+
+1.- Open the terminal window and clone the repository using this command:  
+`https://github.com/41hulk/sclrby.git` 
+
+2.- Change the directory to the react-bookstore directory  
+`cd sclrby`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`   
+4.- In you terminal, run `irb`, following this sequence: copy and paste the person.rb, student.rb, teacher.rb code, then you can check the results  
+
+## Author
+👤 **Ntare Guy**
+- GitHub: [Ntare Guy](https://github.com/41hulk)
+- Twitter: [@Ntare Guy](https://twitter.com/ntare_guy)
+- [LinkedIn](https://www.linkedin.com/in/ntare-guy/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/41hulk/sclrby/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- A special thank for @fernandorpm for this amazing [README template](https://github.com/microverseinc/readme-template)
+
+## 📝 License
+
+This project is [MIT](/LICENSE) licensed.
```

Comment:
- [ ] **[OPTIONAL]** Kindly fix the MIT license link because it shows a `404 page not found error`. Fixing this will make your Readme look more professional 😃.